docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -1,211 +1,172 @@
+- [RectangleArena](object-floors.md#rectanglearena): define the only object you see so far in this scene.
+
+Each node has some customizable properties called **Fields**.
+Let's modify these fields to change the rectangle arena:
+
+> **Hands on #3**: Double-click on the `RectangleArena` node in the scene tree.
+This should open the node and display its fields.
+Select the `floorTileSize` field and set its value to `0.25 0.25` instead of `0.5 0.5`.
+You should see the effect immediately in the 3D view.
+Select the `wallHeight` field and change its value to `0.05` instead of `0.1`.
+The wall of the arena should now be lower.
+
+In the Scene Tree view, the fields are displayed in a different color (depending on the theme) if they differ from their default values.
+Now, we would like to add some objects:
+
+> **Hands on #4**: Double-click on the `RectangleArena` in the scene tree to close it and select it.
+Click on the `Add` button (plus sign) at the top of the scene tree.
+In the open dialog box, choose `PROTO (Webots) / objects / factory / containers / WoodenBox (Solid)`.
+A big box should appear in the middle of the arena.
+Double-click on it in the scene tree to open its fields.
+Change its `size` to `0.1 0.1 0.1` instead of `0.6 0.6 0.6`.
+Change its `translation` to `0 0.05 0` instead of `0 0.3 0`.
+Alternatively, you may use the green arrow that appears in the 3D view to adjust its `translation.y` field.
+Now shift-click and drag the box in the 3D view and move it in some corner of the arena.
+Select the box and press Ctrl-C, Ctrl-V to copy and paste it.
+Shift-click and drag the new box to move it at some different location.
+Create a third box this way.
+Move the boxes, so that no box is at the center of the arena.
+You may also use the green rotation arrows to rotate the boxes along the vertical axis.
+This can be done also by shift-click and drag with the right mouse button.
+Alternatively, you can change the angle of the `rotation` field of the `WoodenBox` nodes in the scene tree.
+Once you are satisfied with the result, save the world using the save button.
+
+Using the translation and rotation handles to move objects is explained in [this section](the-3d-window.md#axis-aligned-handles).
 
 ### Add an e-puck Robot
 
 The e-puck is a small robot having differential wheels, 10 [LEDs](../reference/led.md), and several sensors including 8 [DistanceSensors](../reference/distancesensor.md) and a [Camera](../reference/camera.md).
 In this tutorial we are only interested in using its wheels.
-We will learn how to use some other e-puck features in the other tutorials.
+We will learn how to use other capabilities in the next tutorials.
 
 Now, we are going to add an e-puck model to the world.
 Make sure that the simulation is paused and that the virtual time elapsed is 0.
+If this is not the case, reset the simulation with the `Reset` button (rewind).
```

Comment:
https://cyberbotics.com/doc/guide/the-user-interface?version=fix-tutorial-1#file-menu